lib/plugins/aws/deploy/lib/checkForChanges.js

```diff
@@ -186,7 +188,9 @@ module.exports = {
           this.serverless.service.provider.shouldNotDeploy = true;
 
           const message = ['Service files not changed. Skipping deployment...'].join('');
-          this.serverless.cli.log(message, 'Serverless', { color: 'orange' });
+          legacy.log(message, 'Serverless', { color: 'orange' });
+          progress.get('main').remove();
```

Comment:
I think this line should be removed (as we cannot really remove main progress)

Reply:
good catch, it's was a leftover and it should not be there